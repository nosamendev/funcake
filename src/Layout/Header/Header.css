header {
	position: relative;
	max-width: 1200px;
	margin: 0 auto;
	background-image: url(../../images/header_bg.png);
	background-repeat: no-repeat;
	background-size: cover;
	background-position: center;
	height: 397px;
    text-align: center;
    padding: 0;
	display: block;
	box-sizing: border-box;
}

.logo {
	display: inline-block;
	background: #fff url(../../images/logo.png) center no-repeat;
	text-decoration: none;
	border-radius: 50%;
	position: absolute;
	left: 0;
	top: 0;
}

.welcome {
	max-width: 608px;
	min-height: 504px;
	border-radius: 50%;
	position: relative;
	top: 253px;
	margin: 0 auto;
	background: #fff url(../../images/welcome.png) center 28px no-repeat;
	position: relative;
	text-align: center;
}

.welcome > div {
	max-width: 486px;
	position: relative;
	top: 125px;
	margin: 0 auto;
	text-align: justify;
	padding: 0 20px;
}

@media all and (max-width: 515px) {
	.welcome {
		border-radius: 0;
	}

	.logo {
		width: 120px;
		height: 120px;
		background-size: contain
	}
}

@media all and (min-width: 516px) {
	.logo {
		width: 168px;
		height: 168px;
	}
}

@media all and (max-width: 1055px) {
	.welcome > div {
		top: 195px;
		max-width: none;
	}
}

@media all and (min-width: 1056px) {
	.welcome > div {
		max-width: 486px;
		top: 125px;
	}
}