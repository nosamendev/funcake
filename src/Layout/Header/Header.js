import React from 'react';
import './Header.css';
import { Link } from 'react-router-dom';

const Header = () => {
    return (
        <header>
        	<Link to="/" className="logo"></Link>
			<div className="welcome">
            	<div>Our menu includes trusted signature flavors as well as seasonal and monthly favorites all made fresh daily using local, real ingredients!</div>
            </div>
		</header>
    );
}

export default Header;