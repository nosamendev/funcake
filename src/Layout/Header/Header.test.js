import React from 'react';
import renderer from 'react-test-renderer';
import { Link } from 'react-router-dom';
import Header from './Header';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter});


let wrapper;

beforeEach(() => {
  wrapper = shallow(<Header />);
});

describe('Render Header', () => {
    
    it('render correctly Header component', () => {  
        expect(wrapper).toMatchSnapshot();                                        
    });
});