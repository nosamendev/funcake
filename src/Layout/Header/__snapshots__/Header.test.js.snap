// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header render correctly Header component 1`] = `
<header>
  <Logo />
  <div
    className="welcome"
  >
    <div>
      Our menu includes trusted signature flavors as well as seasonal and monthly favorites all made fresh daily using local, real ingredients!
    </div>
  </div>
</header>
`;
